content/terraform-plugin-framework/v1.15.x/docs/plugin/framework/resources/identity.mdx

```diff
@@ -0,0 +1,194 @@
+---
+page_title: Identity
+description: >-
+Learn how to implement resource identity in the Terraform plugin framework.
+---
+
+# Resource identity
+
+A resource identity is a data object, determined by the provider, that is stored alongside the resource state to uniquely identify a remote object. A resource identity should have the following properties:
+- The resource identity must correspond to at most one remote object per provider, across all instances of that provider.
+- Given a resource identity (during `import`), the provider must be able to determine whether the corresponding remote object exists, and if so, return the resource state.
+- Unless the resource identity schema is being upgraded, the identity data must not change during the lifecycle of the remote object.
+- That is, from the creation of the remote object in the remote system until its destruction.
+
+## Resource Identity Schema
+
+To define the identity object for a managed resource, a resource identity [schema](/terraform/plugin/framework/handling-data/schemas) is provided that consists of a map of attribute names and associated behaviors. This is similar to the existing resource state schema, with a few key differences:
+
+1. The identity schema only supports primitive types and list types such as:
+    - `BoolAttribute`
+    - `Float32Attribute`
+    - `Float64Attribute`
+    - `Int32Attribute`
+    - `Int64Attribute`
+    - `NumberAttribute`
+    - `StringAttribute`
+    - `ListAttribute`
+        - `ElementType: types.BoolType`
+        - `ElementType: types.Float32Type`
+        - `ElementType: types.Float64Type`
+        - `ElementType: types.Int32Type`
+        - `ElementType: types.Int64Type`
+        - `ElementType: types.NumberType`
+        - `ElementType: types.StringType`
+
+<Note>
+
+    The ordering in lists is expected to be stable and semantically relevant.
+
+</Note>
+
+2. Unlike the resource state schema, the resource identity schema will not support behaviors such as Required and Computed. Two behaviors will be allowed for each attribute: `RequiredForImport` and `OptionalForImport`, which, as named, only affect behavior when importing a resource by identity.
```

Comment:
Always use markdown numbers in lists (`1.`) to avoid having to manually number things later. 
```suggestion
1. Unlike the resource state schema, the resource identity schema does not support behaviors such as `Required` and `Computed`. Two behaviors are allowed for each resource state schema: `RequiredForImport` and `OptionalForImport`, which affect behavior when importing a resource by identity.
```